tests/e2e/cluster/up.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(dirname "$(readlink -f "$0")")"
+
+source "${script_dir}/../lib.sh"
+
+# Run kubeadm init and KUBECONFIG is exported on success.
+#
+init_kubeadm() {
+	local kubeadm_config_file="/etc/kubeadm/kubeadm.conf"
+	# Bootstrap the control-plane node.
+	kubeadm init --config "${kubeadm_config_file}"
+
+	export KUBECONFIG=/etc/kubernetes/admin.conf
+
+	# TODO: if we want to run as a regular user.
+	#mkdir -p $HOME/.kube
+	#sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+	#sudo chown $(id -u):$(id -g) $HOME/.kube/config
+
+	# TODO: wait node to show up
+	#kubectl get nodes
+}
+
+# Configure the cluster network with flannel.
+#
+configure_flannel() {
+	kubectl apply -f /opt/flannel/kube-flannel.yml
+
+	check_pods_are_ready "kube-flannel"
+	# TODO: check coredns is up and running too
+}
+
+main() {
+	init_kubeadm
+	configure_flannel
+
+	# TODO: wait node to be ready
+	#
+
+	# Untaint the control-plane node so that pods can be scheduled on it.
+	#local taint="$(kubectl get node/$(hostname) -o jsonpath='{.spec.taints[*].effect}')"
+	#if [ "$taint" == "NoSchedule" ]; then
+		kubectl taint nodes "$(hostname)" \
+			node-role.kubernetes.io/master:NoSchedule-
```

Comment:
I have been using the role `node-role.kubernetes.io/control-plane-` and/or `node-role.kubernetes.io/master-` depending on Kubernetes version.

Reply:
yeah, that piece is broken, I have manually untainted both...I will need to fix it. Thanks!